### ISSUE TITLE
Predictive Services RoadMap Visual

### ISSUE THREAD

**Issue by jakemorr** [bcgov/wps#368]
**Describe the task**
distill the information from Jo's Vision and Goals Miro Board into a visual that can be consumed by stakeholders and other folks that aren't necessarily "in the Know" of our products

**Acceptance Criteria**
- [x] designed for WF1 stakeholders and our executive stakeholders
- [x] brief description of each product we are building
- [x] steps we will take building those products
- [x] a sense of time without any hard dates (small products take up less space on the map) 
- [x] show what has been done to this point
- [x] PDF or Image for sharing

**Additional context**
- Add any other context about the task here.
- Or here

**Comment by PrakritiM** [comment]
[https://app.zenhub.com/files/235861506/6b8a3351-c065-4220-b583-05cd6f993672/download](https://app.zenhub.com/files/235861506/6b8a3351-c065-4220-b583-05cd6f993672/download)

**Comment by PrakritiM** [comment]
Product Phase, however it was decided that updating this file would be an added task.


[https://app.zenhub.com/files/235861506/0e3c3283-51a9-4362-8086-edea98b85bf1/download](https://app.zenhub.com/files/235861506/0e3c3283-51a9-4362-8086-edea98b85bf1/download)

**Comment by PrakritiM** [comment]
![Product Roadmap.jpg](https://images.zenhubusercontent.com/5e331de62bd28c17a1a7b929/14607d14-71e3-40a3-b949-abfa5a3e9540)![Phases.jpg](https://images.zenhubusercontent.com/5e331de62bd28c17a1a7b929/49fddf7d-b3be-424a-a850-74da5e6aeb84)

**Comment by jowand** [comment]
@PrakritiM  I really like the format, I think it's clear what we're doing and where we're going. Ideally when we get some feedback from 90th Percentile Calc we can add in some figures in there in little bubbles or whatever works - e.g. 80% of users affirm the web app saves them time....we have 200 users of the webapp that sort of thing.

Oh damn I wrote you a message just then in Teams but that's a fail... what I said was:
I'm reading through the PSU roadmap visual for #368. I've added comments in the attached pdf
